Banner/DutchFlag.dfy

```diff
@@ -0,0 +1,43 @@
+method Main()
```

Comment:
Could you please re-take the screenshot? Looks like you have an outdated version of VSCode because I fixed the bug that highlights 0<=i<j<a.Length in blue.

Reply:
Done